cli/tests/pcluster/config/test_config_patch.py

```diff
@@ -391,6 +391,90 @@ def _test_queues(base_conf, target_conf):
     )
 
 
+def _test_no_updatable_custom_actions(base_conf, target_conf):
+    base_conf["HeadNode"].update(
+        {
+            "CustomActions": {
+                "OnNodeStart": {"Script": "test-to-remove.sh"},
+                "OnNodeConfigured": {"Script": "test-to-edit.sh", "Args": ["1", "2"]},
+            }
+        }
+    )
+    target_conf["HeadNode"].update(
+        {"CustomActions": {"OnNodeConfigured": {"Script": "test-to-edit.sh", "Args": ["2"]}}}
+    )
+
+    _check_patch(
+        base_conf,
+        target_conf,
+        [
+            Change(
+                ["HeadNode", "CustomActions"],
+                "OnNodeStart",
+                {"Script": "test-to-remove.sh"},
+                "-",
+                UpdatePolicy.UNSUPPORTED,
+                is_list=False,
+            ),
+            Change(
+                ["HeadNode", "CustomActions", "OnNodeConfigured"],
+                "Args",
+                ["1", "2"],
+                ["2"],
+                UpdatePolicy.UNSUPPORTED,
+                is_list=False,
+            ),
+        ],
+        UpdatePolicy.UNSUPPORTED,
+    )
+
+
+def _test_updatable_custom_actions_attributes(base_conf, target_conf):
+    base_conf["HeadNode"].update(
+        {"CustomActions": {"OnNodeUpdated": {"Script": "test-to-edit.sh", "Args": ["1", "2"]}}}
+    )
+
+    target_conf["HeadNode"].update(
+        {"CustomActions": {"OnNodeUpdated": {"Script": "test-edited.sh", "Args": ["1", "2"]}}}
```

Comment:
I'd change args as well, just to add an extra "check".

Reply:
good catch, will do